pallets/staking/src/lib.rs

```diff
@@ -1837,6 +1837,19 @@ pub mod pallet {
             Ok(())
         }
 
+        #[pallet::weight(T::WeightInfo::undelegate())]
+        pub fn force_undelegate(
+            origin: OriginFor<T>,
+            accounts: Vec<T::AccountId>,
+        ) -> DispatchResult {
+            ensure_root(origin)?;
+            for account in accounts {
+                Self::_undelegate(&account);
```

Comment:
> TODO: for those not continue delegating's account, also need slash credit
Do we need to slash credit here?

Reply:
should slash credit. But also need slash credit for  not online.